internal/integration/testcase_test.go

```diff
@@ -51,11 +51,17 @@ func (t testcase) Init() testcase {
 
 	if t.PipelineName == "" {
 		namePrefix := t.Name()
-		jobID := os.Getenv("BUILDKITE_JOB_ID")
-		if jobID == "" {
-			jobID = strconv.FormatInt(time.Now().UnixNano(), 10)
+		buildNum := os.Getenv("BUILDKITE_BUILD_NUMBER")
+		if buildNum == "" {
+			buildNum = strconv.FormatInt(time.Now().UnixNano(), 10)
+		}
+
+		t.PipelineName = strings.ToLower(fmt.Sprintf("test-%s-%s", namePrefix, buildNum))
+
+		retryNum := os.Getenv("BUILDKITE_RETRY_COUNT")
+		if retryNum != "0" {
+			t.PipelineName = fmt.Sprintf("%s-retry-%s", t.PipelineName, retryNum)
```

Comment:
Ok looks good. The reason I chose job id is that it's guaranteed to be unique. This is too, I guess. I usually just copy the job id from the URL anchor when I need. But this should be easier to search for on the page.

Reply:
yeah, i also changed it so because the UUIDs are really long and non-human-readable, and get truncated on my builds page:
![CleanShot 2024-02-28 at 16 08 12@2x](https://github.com/buildkite/agent-stack-k8s/assets/15753101/923e519e-09ea-4f89-80a0-d789058a5a8f)